apps/web/src/pages/TalentNominations/NominationGroup/components/ExperienceByTypeAccordion.tsx

```diff
@@ -0,0 +1,68 @@
+import { Accordion, CardBasic } from "@gc-digital-talent/ui";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { Experience } from "@gc-digital-talent/graphql";
+
+import ExperienceCard from "~/components/ExperienceCard/ExperienceCard";
+
+interface ExperienceByTypeAccordionProps {
+  experienceSections: {
+    id: string;
+    title: string;
+    experiences: Omit<Experience, "user">[];
+  }[];
+  openSections: string[];
+  defaultOpen?: boolean;
+  setOpenSections: (sections: string[]) => void;
+}
+
+const ExperienceByTypeAccordion = ({
+  experienceSections,
+  openSections: expandedItems,
+  setOpenSections,
+}: ExperienceByTypeAccordionProps) => {
+  return (
+    <Accordion.Root
+      type="multiple"
+      mode="card"
+      value={expandedItems}
+      onValueChange={setOpenSections} // Sync state with Accordion
+      data-h2-margin="base(0, 0)"
+    >
+      {experienceSections
+        .filter(
+          ({ experiences: sectionExperiences }) =>
+            sectionExperiences.length > 0,
+        )
+        .map(({ id, title, experiences: sectionExperiences }) => (
+          <Accordion.Item key={id} value={id}>
+            <Accordion.Trigger>
+              {/* eslint-disable-next-line */}
```

Comment:
Can we be specific about the disable?
```suggestion
              {/* eslint-disable-next-line formatjs/no-literal-string-in-jsx */}
```

Reply:
This is because the string that comes here already translated and doesn't need translation here.